modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/actionItems/filterCreationActions.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ICreationActionItem} from '../../management_bar/components/CreationMenu';
+
+const filterCreationActions = ({creationActions, globalCollectionActions}: {
+	creationActions: Array<ICreationActionItem>,
+	globalCollectionActions: any
+}): Array<ICreationActionItem> | null => {
+	return creationActions
+		? creationActions.reduce(
```

Comment:
[not blocking] I've always found `reduce` to be rather hard to read. For a `filter*` method, maybe JS `filter` would be a cleaner fit. This is just my code style opinion though, this is fine as it is.